packages/sanity/src/desk/comments/src/types.ts

```diff
@@ -0,0 +1,187 @@
+import {PortableTextBlock, User} from '@sanity/types'
+
+/**
+ * @beta
+ * @hidden
```

Comment:
would be good if we could add a todo here that calls out documenting this as a remaining task for this feature

Reply:
I am not sure I understand, can you elaborate? :)

Reply:
Even if it's considered internal/hidden for now, I think having some rudimentary api docs would be great and also helpful for us internally :)